docs/components/modeler/web-modeler/deploy-process-application.md

```diff
@@ -30,20 +43,23 @@ All BPMN, DMN, and form files contained in the process application folder are de
 If any resource fails to deploy, the whole deployment [fails](#deployment-errors) and the cluster state remains unchanged. This safely ensures that a process application cannot be deployed incompletely or in an inconsistent state.
 :::
 
-## Run a process application
+## Run your process application
+
+You can also manually [run](/components/modeler/web-modeler/run-or-publish-your-process.md#run-a-process) your process application if required, for example to perform manual testing.
```

Comment:
Same sort of clarification/context here -

"To test a process once it has been deployed to a testing, staging, or production environment, use the **Run** option. Using Run once a process is deployed allows you to observe how your process performs in a live environment, and not only with test data provided using Play."

Reply:
Thanks, feedback incorporated 👍